### ISSUE TITLE
select multiple files/images

### ISSUE THREAD

**Issue by darkralts** [krille-chan/fluffychat#725]
### Feature Description

in element and schildichat you can send as many images as you want all at once itl send them one by one but you can selectthem, its tedious not able to do that in fluffychat

### Rationale

ease of use intuitiveness making everything easier, not having to go through the menus over and over if you wanna send a friend like 10 images

**Comment by bombermuecke** [comment]
It is also very important for me to be able to send several pictures at once. I would therefore be very happy if this feature were added.

**Comment by t0maz** [comment]
Yep, this one is holding our family migration to Matrix clients due to they lack of ability to easily select and send multiple media files from iOS "share" contect menu. This is most used feature of IM apps by us as we share a lot photos and movies.
Till then we stick to XMPP Monal client which support this feature.
Still we do really hope to see it coming to Fluffy to!

**Comment by Janavanwalde** [comment]
It's already installed :)

**Comment by darkralts** [comment]
Noticed it working great after updating, this client has fixed many bugs and added good features in the last few months
